sigstore-java/src/main/java/dev/sigstore/oidc/client/OidcClients.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2023 The Sigstore Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package dev.sigstore.oidc.client;
+
+import com.google.common.collect.ImmutableList;
+import java.util.List;
+
+/** An ordered list of oidc clients to use when looking for credentials. */
+public class OidcClients {
+
+  public static final OidcClients DEFAULTS =
+      new OidcClients(
+          ImmutableList.of(
+              GithubActionsOidcClient.builder().build(), WebOidcClient.builder().build()));
+
+  public static final OidcClients STAGING_DEFAULTS =
+      new OidcClients(
+          ImmutableList.of(
+              GithubActionsOidcClient.builder().build(),
+              WebOidcClient.builder().setIssuer(WebOidcClient.STAGING_DEX_ISSUER).build()));
```

Comment:
It could delegate to of(...)

It might be helpful to add vararg-based of(...) factory

Reply:
true true, lemme just use varargs.

Reply:
I see you've added varargs, however, you have not start using them :-/